InstanceExport/PowerShell/README.md

```diff
@@ -27,8 +27,63 @@ The process will prompt to you the mandatory fields to be typed in your screen `
 It will run automatically after you enter the fields and it will prompt the progress of your data been exported by each available category. When all is finished you should see the message "Exporting Instance finished".
 
 The created power shell script has five parameters that are explained below:
+
 - manifestFilePath: Path (or just file name, if same directory) of the manifest file you have downloaded using step 2, e.g. `C:\Users\MyUser\InstanceExport\manifest.json`.
 - exportFilePath: Path (or just file name, if same directory) to create a folder for the exported files, e.g. `C:\Users\MyUser\InstanceExport\2021_Export\`; you do not need to create this folder manually, the script will do so for you.
 - userName: Your username (work email) for login at DevResults, e.g. `first.last@org.org`.
 - password: Your password for login at DevResults or API Key's Secret.
 - overwrite: Optional parameter to inform if you want to overwrite files that already exist and replace them. If you don't use the value of it will be false, which means that if a file already exists in the exportFilePath it will be skipped.
```

Comment:
I don't get prompted for the overwrite parameter - should we take it out? how do you set it?

Reply:
Hey RP,
You can use the overwrite parameter like I'm showing in the image below but you will need to create a `SecureString` for your password
![image](https://user-images.githubusercontent.com/67288628/225468832-d4fc83d7-4980-45b4-8a69-094f17e67b0d.png)